### ISSUE TITLE
Update Electron

### ISSUE THREAD

**Issue by bradrn** [HeinrichApfelmus/threepenny-gui#246]
`threepenny-gui`’s recommended [`package.json`](https://github.com/HeinrichApfelmus/threepenny-gui/blob/9b891a328406655265f0fbdd371124ef2654f998/doc/electron/package.json) uses Electron version `^1.6.1`. By contrast, the Electron quickstart [`package.json`](https://github.com/electron/electron-quick-start/blob/master/package.json) uses Electron version `^10.1.3` — nine major versions later! It looks like `threepenny-gui` desperately needs to upgrade its Electron version. To do this, the package versions in [`package.json`](https://github.com/HeinrichApfelmus/threepenny-gui/blob/9b891a328406655265f0fbdd371124ef2654f998/doc/electron/package.json) would need to be upgraded; after that the [provided `electron.js`](https://github.com/HeinrichApfelmus/threepenny-gui/blob/9b891a328406655265f0fbdd371124ef2654f998/doc/electron/electron.js) will probably need to be changed as well, in order to keep it working with the updated package versions.

(This seems like a simple change, and I would submit a PR myself, except that (a) my knowledge of the node.js ecosystem is practically nonexistent, and (b) I [cannot build `threepenny-gui`](https://github.com/commercialhaskell/stack/issues/5404), and thus am unable to test any changes I may make. If by some miracle I manage to get everything working again, I can certainly try to submit a PR for this, but until then I can’t really do anything with `threepenny-gui`.)

**Comment by sjakobi** [comment]
Unfortunately I'm also quite unfamiliar with Electron and the node ecosystem. I'd appreciate a PR!

**Comment by adi-mohan** [comment]
Hi! May I work on this issue?

**Comment by adi-mohan** [comment]
Also, you should almost never use electron in dependencies. It should be under devDependencies, reason being electron is not advised to install on production systems. I will initiate PR, whenever you respond to this. @bradrn @HeinrichApfelmus @sjakobi

**Comment by bradrn** [comment]
I’d be happy for you to work on this issue, although it would be nice to have some comment from @HeinrichApfelmus on whether that’s OK (which I assume it is).

**Comment by adi-mohan** [comment]
For the time being I will create PR and check if it passes CI. @HeinrichApfelmus can see the results and then take the decision.

**Comment by HeinrichApfelmus** [comment]
@adi-mohan Yes, please! I'm not very familiar with the `npm` ecosystem, so any help is appreciated.

**Comment by bradrn** [comment]
Update on this issue: I’ve been working recently on a threepenny-gui application in Electron, and I think I’ve managed to get it working with the latest versions of everything. I’d like to do a rewrite of the existing tutorial to get it a bit more up-to-date, but I think I’ll just wait a bit until I’m confident I’ve gotten everything working properly.
